react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -246,7 +247,7 @@ class Thumbnail extends PureComponent<Props> {
                     { audioMuted && !_isVirtualScreenshare && <AudioMutedIndicator /> }
                     { !tileView && _pinned && <PinnedIndicator />}
                     { renderModeratorIndicator && !_isVirtualScreenshare && <ModeratorIndicator />}
-                    { !tileView && (isScreenShare || _isVirtualScreenshare)
+                    { !tileView && ((isScreenShare && !_isMultiStreamSupportEnabled) || _isVirtualScreenshare)
```

Comment:
what's happening here? the old logic made sense:

"show the indicator either when it's a screen share or a virtual screen share participant"

now it says:

"show the indicator when it's a screen share and no multi stream enabled or when it's a virtual screen share"

why the case for "screen share and not multi stream enabled"? if it's about local screen share then the code would be self documenting if it included the local flag in the condition otherwise it's worth putting a comment

Reply:
`screen share and not multi stream enabled` is the case for the local camera thumbnail in the case when multi-stream is enabled but is not supported by the endpoint like RN. In this case, `isScreenShare` will be true. So in the case where multi-stream flag is not enabled, there will be only one thumbnail and we want to show the screensharing indicator there. When multi-stream flag is enabled, there will be a separate thumbnail for SS and we don't want to show the screensharing indicator on the camera thumbnail.